### PR TITLE
AutoGluon MultiModal model SageMaker deploy tutorial

### PR DESCRIPTION
**Pull request by gidler**
*Description of changes:*
AutoGluon Deep Learning Containers MultiModal SageMaker Training and Inference Tutorial

*Testing done:*
Ran notebook end-to-end.

## Merge Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your pull request._

- [x] I have read the [CONTRIBUTING](https://github.com/aws/amazon-sagemaker-examples/blob/master/CONTRIBUTING.md) doc and adhered to the example notebook best practices
- [x] I have updated any necessary documentation, including [READMEs](https://github.com/aws/amazon-sagemaker-examples/blob/master/README.md)
- [x] I have tested my notebook(s) and ensured it runs end-to-end
- [x] I have linted my notebook(s) and code using `black-nb -l 100 {path}/{notebook-name}.ipynb`

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: advanced_functionality/autogluon-multimodal-containers/scripts/multimodal_serve.py [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+from PIL import Image
+
+from autogluon.multimodal import MultiModalPredictor
+
+
+def _decode_and_save_base85_image(image_b85):
+    """saves Base65 encoded image to disk and returns unique name"""
+
+    image_bytes = base64.b85decode(image_b85)
+    image_hash = hashlib.md5(image_bytes).hexdigest()
```
[the remark this answers is not in the document]
Comment (gidler) [reply advanced_functionality/autogluon-multimodal-containers/scripts/multimodal_serve.py:16]: MD5 is suitable for this non-cryptographic purpose and preferable over more recent secure hash algorithms due to lower computational requirements. A maliciously design hash collision is acceptable in this use case.

[the remark this answers is not in the document]
Comment (gidler) [reply advanced_functionality/autogluon-multimodal-containers/scripts/multimodal_serve.py:16]: MD5 is suitable for this non-cryptographic purpose and preferable over more recent secure hash algorithms due to lower computational requirements ([see here](https://en.wikipedia.org/wiki/MD5)). A maliciously designed hash collision is acceptable in this use case.

**Comment by gidler** [comment]
Repo maintainers, I have 2 questions on this PR:

1. I see the AWS CodeBuild CI Report Result is `FAILED`, but the links for `Details` and `Build logs` don't take me anywhere. How do I determine what is causing the failure?
2. This tutorial is similar to the `advanced_functionality/autogluon-tabular-containers` tutorial. There is one class `AutoGluonTraining` in `ag_model.py` that is exactly the same in both examples. Would you prefer contributors to minimize duplicate code between tutorials by re-using classes across example directories, or would you prefer that each example be entirely self-contained?

**Comment by zhaoqizqwang** [comment]
Closing this PR since it was created a long time ago and didn't pass all the checks. Feel free to open a new pull request instead.

**Pull request closed without merging**